clients/validator/src/index.ts

```diff
@@ -61,12 +65,12 @@ export default class ValidatorClient {
         // if we have more than a single validator, try to perform initial connection until we succeed or run out of options
         if (validatorUrls.length > 1) {
             for (let i = 0; i < validatorUrls.length; i++) {
-                console.log("Attempting initial connection to", validatorUrls[0])
-                const netClient = await NetClient.connect(wallet, validatorUrls[0], stakeDenom).catch((_) => ValidatorClient.moveArrayHeadToBack(validatorUrls))
+                console.log("Attempting initial connection to", validatorUrls[i])
```

Comment:
It should be reading from [0] all the time, not [i] as on failure the head of the validators array will be moved to the back.  This was a total overcomplication on my end and if somebody refactors it at some point, it will be great (but I don't think it should be done in this PR)

Reply:
Ah alright, it caught my eye on that 0, but it make sense, I'll just leave it as is for now.